Updated configuring realms chapter #1483 (#1558)
keycloak/keycloak-documentation: server_admin/topics/login-settings/forgot-password.adoc

== Enabling forgot password

If you enable `Forgot password`, users can reset their login credentials if they forget their passwords or lose their OTP generator. 

.Procedure
ifeval::[{project_product}==true]
. Click *Realm Settings* in the menu.
endif::[]
ifeval::[{project_community}==true]
. Click *Realm settings* in the menu.
endif::[]
. Click the *Login* tab.
+
.Login tab

> image:{project_images}/login-tab.png[Login Tab]

Reviewer: This image is the old version.

Author: Thanks @andymunro , I have updated the image.

> image:{project_images}/login-tab.png[Login Tab]

Reviewer: This screen shot is for the new console, but it also appears in the old console documentation.

image:{project_images}/reset-credentials-flow.png[Reset Credentials Flow]
+
If you do not want to reset the OTP, set the `Reset OTP` requirement to *Disabled*.
ifeval::[{project_product}==true]
. Click the *Required Actions* tab. Ensure _Update Password_ is enabled.
endif::[]
ifeval::[{project_community}==true]

> . Navigate back to Authentication and click the *Required actions* tab. Ensure _Update Password_ is enabled.
> image:{project_images}/reset-credentials-required-actions.png[Reset Credentials Required Actions]

Reviewer: ```suggestion
. Click *Authentication* in the menu.
. Click the *Required actions* tab. 
. Ensure *Update Password* is enabled.
+
.Required Actions
image:{project_images}/reset-credentials-required-actions.png[Required Actions]
```
